Makefile

```diff
@@ -103,8 +103,10 @@ dev-env-client:
 
 .PHONY: dev-env-server
 dev-env-server:
+	source server/install-mac-m1-reqs.sh
```

Comment:
this automates the installation of pip pkgs that currently require some special handling/ordering on a M1